.env.example

```
@@ -0,0 +1,15 @@
+DATABASE_URL=postgresql+psycopg://postgres:postgres@postgres:5432/smartvault
+REDIS_URL=redis://redis:6379/0
+
+# --- App Settings ---
+APP_ENV=development
+DEBUG=true
+
+# --- Secrets (REQUIRED) ---
+# You must set these yourself.
+# Run: openssl rand -hex 32
+SECRET_KEY=
+API_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)